tests/application/static_site/test_views.py

```diff
@@ -763,7 +763,7 @@ class TestMeasurePage:
     @pytest.mark.parametrize(
         "static_mode, expected_url",
         (
-            # ("yes", "<some_url>"),
+            ("yes", "/topic/subtopic/measure/latest/downloads/dimension-title-table.csv"),
```

Comment:
Can we do away with the comment above now? And maybe I was right about the static build being the reason for those weird url formations?

Reply:
Good spot 👍